_pages/learning/crystal_programming.html

```diff
@@ -1,81 +1,124 @@
 ---
-title: "Book: Crystal Programming"
-description: |
-  <strong>Crystal Programming</strong> is a project-based book by George Dietrich and Guilherme Bernal that covers all the
-  fundamental concepts needed to start programming in Crystal, such as OOP and concurrent programming.
-layout: learning
+title: "Building efficient, safe, and readable web and CLI applications with Crystal"
+layout: wide
+---
 
-headline: Building efficient, safe, and readable web and CLI applications with Crystal
-cover_image: /assets/learning/crystal_programming.png
-vendors:
-- url: https://www.amazon.com/Crystal-Programming-project-based-introduction-applications/dp/1801818673/ref=sr_1_2
-  name: Amazon
-- url: https://www.packtpub.com/product/crystal-programming/9781801818674
-  name: Packt
+<div class="book-presenter">
+  <div class="book-presenter__main">
```

Comment:
The nested main div seems unnecessary.
Other places use a grid layout with template areas for this. E.g. `grid-template-areas: "cover description" "body body"`. This keeps the html minimal and semantic.